### DISCUSSION TITLE
Is it safe to use PocketBase for production?

### DISCUSSION THREAD

**Discussion by shar0** [pocketbase/pocketbase#2860] [Q&A] [answered]
Hi, 
PocketBase seems to almost perfectly fit my needs. 
If I start using PocketBase now for developing my web app as Go framework, 
and minimally use basic functionalities and the web admin, 
will it be possible to migrate to it once the version 1.0 is released?

**Comment by ganigeorgiev** [comment, marked as answer]
I cannot make any promises.

Although I try to avoid breaking the exposed APIs, full backward compatibility is no guarantee and you may have to read the [CHANGELOG](https://github.com/pocketbase/pocketbase/blob/master/CHANGELOG.md) and apply some manual migration steps from time to time.

**Reply by arnavmehta7** [reply]
@ganigeorgiev I have found the documentation for using `as a framework` in go to be outdated and incomplete. Can it please be updated as well?
Thanks!

**Reply by ganigeorgiev** [reply]
@arnavmehta7 The Go guides will be updated with the next release.

**Reply by arnavmehta7** [reply]
@ganigeorgiev one more question, is there any ETA when we'll have a stable release?

**Reply by ganigeorgiev** [reply]
Sorry, there are no hard ETAs. If you haven't already, you can check the current active tasks in the [public roadmap](https://github.com/orgs/pocketbase/projects/2/views/1).
After the upcoming major v0.17.0 release the SDKs refactoring will be prioritized and will slowly start dedicating slightly more time towards v3 of my other open source project ([Presentator#183](https://github.com/presentator/presentator/issues/183)).
